test/unit/operations/third_party/iosxr/test_rpc.py

```diff
@@ -0,0 +1,23 @@
+from ncclient.operations.third_party.juniper.rpc import *
+import json
+import unittest
+from mock import patch
+from ncclient import manager
+import ncclient.manager
+import ncclient.transport
+from ncclient.xml_ import *
+from ncclient.operations import RaiseMode
+from xml.etree import ElementTree
+
+
+class TestRPC(unittest.TestCase):
+
+    @patch('ncclient.transport.SSHSession')
+    @patch('ncclient.operations.third_party.iosxr.rpc.RPC._request')
+    def test_execute_rpc(self, mock_request, mock_session):
+        device_handler = manager.make_device_handler({'name': 'junos'})
```

Comment:
Why is a test case, apparently for the IOS XR device handler, referring to the junos device handler?

Reply:
nice catch, I have fixed this in latest commit